Proteomics/Modifications/ModificationLocalization.cs

```diff
@@ -4,10 +4,10 @@ namespace Proteomics
 {
     public static class ModificationLocalization
     {
-        public static bool ModFits(ModificationWithMass attemptToLocalize, Protein protein, int peptideOneBasedIndex, int peptideLength, int proteinOneBasedIndex)
+        public static bool ModFits(ModificationGeneral attemptToLocalize, Protein protein, int peptideOneBasedIndex, int peptideLength, int proteinOneBasedIndex)
```

Comment:
can we change the name of ModificationGeneral to just Modification

Reply:
that's my plan once everything is merged. problem is that "modification" is in MM and PS. I was worried that if I used the same name, some errors might get in accidentally. It would probably be okay to do it. I was just being cautious